.github/CODEOWNERS

```diff
@@ -1482,7 +1482,7 @@
 ###########
 /.vscode/ @mikeharder @ckairen @deyaaeldeen @jeremymeng
 /common/ @mikeharder @ckairen @deyaaeldeen @jeremymeng
-/common/config/rush/pnpm-lock.yaml @ckairen @jeremymeng @Azure/azure-sdk-js-dev
+/common/config/rush/pnpm-lock.yaml @ckairen @jeremymeng @Azure/azure-sdk-js-dev @qiaozha @MaryGao
```

Comment:
@jeremymeng Not sure this is reason we can't merge our mgmt release pr because of introducing pnpm-lock.yaml change.

https://github.com/Azure/azure-sdk-for-js/pull/30041

![image](https://github.com/Azure/azure-sdk-for-js/assets/9943211/122b5e9d-da56-4929-893e-99f01e466fed)

Reply:
@MaryGao I added you and Qiaoqiao to the azure-sdk-js-dev group